src/software/thunderscope/log/g3log_widget.py

```diff
@@ -82,6 +86,6 @@ def refresh(self):
             )
         ):
             log_str = f"R{log.robot_id} {log.created_timestamp.epoch_timestamp_seconds} {self.log_level_str_map[log.log_level]} [{log.file_name}->{log.line_number}] {log.log_msg}\n"
-            self.console_widget.write(log_str)
+            self.console_widget.repl.write(log_str, style="output")
```

Comment:
Curious what this change is for?

Reply:
we relied on implementation details of ConsoleWidget which changed after updating pyqtgraph